packages/fluent/scss/menu/_theme.scss

```diff
@@ -88,6 +99,12 @@
                 }
             }
         }
+
+        > .k-separator {
+            @include fill(
+                $border: $kendo-menu-separator-border
```

Comment:
Same here add css variable

Reply:
The syntax here should be: $border: var( --kendo-menu-separator-border, #{$kendo-menu-separator-border} )